client/src/components/RatingsAndReviews/WritingReview/WritingReview.jsx

```diff
@@ -28,150 +28,152 @@ class WritingReview extends React.Component {
+              <input type="radio" value='1' name={lengthStr} /> Runs Short
+              <input type="radio" value='2' name={lengthStr} /> Runs slightly short
+              <input type="radio" value='3' name={lengthStr} /> Perfect
+              <input type="radio" value='4' name={lengthStr} /> Runs slightly long
+              <input type="radio" value='5' name={lengthStr} /> Runs long
+              <input type="radio" value='0' name={lengthStr} defaultChecked /> none
           </div>
           }
           {chars.Fit &&
           <div>
-          <p>Fit</p>
-          <input type="radio" value='1' name={fitStr} /> Runs tight
-          <input type="radio" value='2' name={fitStr}/> Runs slightly tight
-          <input type="radio" value='3' name={fitStr} /> Perfect
-          <input type="radio" value='4' name={fitStr} /> Runs slightly long
-          <input type="radio" value='5' name={fitStr} /> Runs long
-          <input type="radio" value='0' name={fitStr} defaultChecked /> none
+              <p>Fit</p>
+              <input type="radio" value='1' name={fitStr} /> Runs tight
+              <input type="radio" value='2' name={fitStr}/> Runs slightly tight
+              <input type="radio" value='3' name={fitStr} /> Perfect
+              <input type="radio" value='4' name={fitStr} /> Runs slightly long
+              <input type="radio" value='5' name={fitStr} /> Runs long
+              <input type="radio" value='0' name={fitStr} defaultChecked /> none
           </div>
            }
        </div>
         <div>
-        <p>Add a headline</p>
-        <textarea
-        name='summary'
-          value={summary}
-          onChange={this.handleChange}
-          placeholder="what's most important to know?"
-          rows={1}
-          cols={50}
-        />
+            <p>Add a headline</p>
+            <textarea
+              name='summary'
+              value={summary}
+              onChange={this.handleChange}
+              placeholder="what's most important to know?"
+              rows={1}
+              cols={50}
+            />
       </div>
       <div>
-        <p>Add a written review </p>
-        <textarea
-        name='body'
-          value={body}
-          onChange={this.handleChange}
-          placeholder="what did you like or dislike? what did you use this product for?"
-          rows={5}
-          cols={50}
-        />
+            <p>Add a written review </p>
+            <textarea
+            name='body'
+              value={body}
+              onChange={this.handleChange}
+              placeholder="what did you like or dislike? what did you use this product for?"
+              rows={5}
+              cols={50}
+            />
       </div>
-        <div>
+      <div>
           <p>your nickname:
             <input name='name' type='text' value={name} onChange={this.handleChange} />
           </p>
-          </div>
-          <div>
+      </div>
+      <div>
             <p>your email:
```

Comment:
If you want to do the email validation. This is how I did on mine 
https://github.com/sensational-sycamore/sycamore/blob/27982cf2c10f045bf74a075abd38202ff10db1cc/client/src/components/QuestionsAndAnswers/AddAnswerModal/AddAnswerModal.jsx#L88

Reply:
added email validation